src/store/messages/saga.ts

```diff
@@ -61,9 +61,24 @@ export function* fetch(action) {
 export function* send(action) {
   const { channelId, message } = action.payload;
 
-  yield call(sendMessagesByChannelId, channelId, message);
+  const messagesResponse = yield call(sendMessagesByChannelId, channelId, message);
+  if (messagesResponse.status === 200) {
```

Comment:
for things like this where an entire block lives inside an `if` statement, i generally prefer either: setting it up as a guard rather than a block, so: 
`if (messagesResponse.status !== 200) return;`

or extracting to a method so it's only a one-liner inside the `if` block.
```
if (messagesResponse.status === 200) {
    yield sendMessage(messageResponse);
}
```

Reply:
also: tests!